web/profiles/joinup/config/install/user.role.authenticated.yml

```diff
@@ -7,8 +7,7 @@ weight: 1
 is_admin: false
 permissions:
   - 'access content'
-  - 'create collection rdf entity'
```

Comment:
You removed this permission. Please, also update the authentication.feature behat tests because it fails while testing for access for the authenticated user.